decidim-proposals/config/locales/en.yml

```diff
@@ -119,6 +127,14 @@ en:
       create:
         error: There's been errors when saving the proposal.
         success: Proposal created successfully.
+      events:
+        update_proposal_category_event:
+          email_intro: |-
+            Hi,
```

Comment:
Remove the "Hi, new line" part, it was an error introduced in previous events!

Reply:
ok, I'll remove it